evaluator/evaluator.py

```diff
@@ -92,57 +94,61 @@ async def _store_new_cves(rh_account_id, system_id, new_cves, conn, advisory_ava
     if cve_metadata_missing:
         new_cves_ids.update(await _register_missing_cves(cve_metadata_missing, conn))
 
-    cve_system_list = []
     for cve_id in new_cves_ids:
         remediation_type_id = get_available_remediation_type(advisory_available, None, None, 0)
-        cve_system_list.append((rh_account_id, system_id, cve_id, advisory_available, remediation_type_id))
-    await conn.executemany("""insert into system_vulnerabilities (rh_account_id, system_id, cve_id, advisory_available, remediation_type_id)
-                              values ($1, $2, $3, $4, $5)""", cve_system_list)
+        row = await conn.fetchrow("""insert into system_vulnerabilities (rh_account_id, system_id, cve_id, advisory_available, remediation_type_id)
```

Comment:
This will generate hundreds of insert statements for each CVE every system and decrease the performance a lot.

Reply:
Yes, I know and I tried to fix this before I submitted this. But it is completely shitty since asyncpg's `execute*` methods discard the query returning values and `fetch*` methods does not accept python iterable as their argument. The only solution I found for this is [this](https://stackoverflow.com/questions/43739123/best-way-to-insert-multiple-rows-with-asyncpg/43939504#43939504). And I also do not like this solution as it requires to put all values in the tuple and not only the needed ones.

Reply:
Any ideas?

Reply:
Using `unnest` for each row member works. https://dev.to/forbeslindesay/postgres-unnest-cheat-sheet-for-bulk-operations-1obg